packages/eslint-plugin/docs/rules/prefer-string-starts-ends-with.md

```diff
@@ -52,6 +52,26 @@ foo.startsWith('bar');
 foo.endsWith('bar');
 ```
 
+<!--/tabs-->
+
+## Options
+
+### `allowSingleElementEquality`
+
+If switched to `'always'`, the rule will allow equality checks against the first or last character in a string.
+This can be preferable in projects that don't deal with special character encodings and prefer a more succinct style.
+
+The following code is considered incorrect by default, but is allowed with `allowSingleElementEquality: 'always'`:
+
+```ts
```

Comment:
```suggestion
```ts option='{ "allowSingleElementEquality": "always" }' showPlaygroundButton
```

What do you think about adding "Open in Playground" button here? :slightly_smiling_face:

Reply:
Oh good point. I wonder if we should add in something to make sure we always remember to ask for that (I don't)... or make it more automatic. 🤔